modules/monix/src/weaver/monixcompat/suites.scala

```diff
@@ -0,0 +1,24 @@
+package weaver
+package monixcompat
+
+import cats.effect.{ ConcurrentEffect, ContextShift, Resource, Timer }
+
+import monix.eval.Task
+import monix.eval.instances._
+import monix.execution.Scheduler
+
+trait BaseMonixSuite { self: ConcurrentEffectSuite[Task] =>
+  val ec                              = scala.concurrent.ExecutionContext.global
```

Comment:
I think I wouldn't expose this one publicly. `Scheduler` is more central to monix, and also extends ExecutionContext

Reply:
👍 I'll replace it with `import monix.execution.Scheduler.Implicits.global`, which I imagine wraps `scala.concurrent.ExecutionContext.global`.

Reply:
I'd just do `val scheduler : Scheduler = monix.execution.Scheduler.global`, no need to refer to it implicitly I think, considering it's not used in combinator methods on Task

Reply:
Addressed in b644c8a823f102f8bcc638299adb09818008ed9c.